components/Menu.tsx

```diff
@@ -0,0 +1,130 @@
+import React, { useState } from 'react'
+import Image from 'next/dist/client/image'
+import { IoCloseSharp, IoMenu } from 'react-icons/io5'
+
+export default function Menu() {
+  const [menuOpen, setMenuOpen] = useState(false)
+
+  const handleToggle = () => {
+    setMenuOpen(!menuOpen)
+  }
+
+  return (
+    <nav className="bg-white m-auto w-full lg:w-11/12 xl:w-10/12 md:rounded-full">
+      <div className="px-6 md:mx-auto md:px-12">
+        <div className="flex justify-between ">
+          <div className="flex space-x-7">
+            <div>
+              <a href="#" className="flex  items-center px-2">
+                <span className="flex font-semibold text-blue-800 text-lg">
+                  <Image
+                    src="/logoCapaniaWide.png"
+                    alt="Logo un techo para mi club"
+                    width="218"
+                    height="71"
+                    className="justify-self-center max-w-md"
+                  />
+                </span>
+              </a>
+            </div>
+          </div>
+          <div className="hidden md:flex items-center space-x-3 ">
+            <div className="hidden md:flex items-center space-x-1">
+              <a
+                href=""
+                className="py-4 px-2 text-lg xl:text-2xl text-blue-500 font-semibold border-blue-500  "
+              >
+                Campaña
+              </a>
+              <a
```

Comment:
What if we encapsulate this `<a>` element as a separated component (it could live in this same file even) to avoid repeating the long `className` property?

It would also be cool if we handle the active link based on the URL. I think [this way ](https://gist.github.com/remy/0dde38897d6d660f0b63867c2344fb59)seems a good way to handle that.

For this first release we are going to have only `/home` and `/ayudanos` , we probably should add "Inicio" as a menu item and hide the rest for now until we develop the pages. That said, maybe the active link is not even a problem now so we could create a different ticket for it.

Reply:
I think is a great idea!